shenyu-admin/src/main/java/org/apache/shenyu/admin/listener/http/HttpLongPollingDataChangedListener.java

```diff
@@ -195,16 +195,16 @@ protected void afterSelectorChanged(final List<SelectorData> changed, final Data
     /**
      * check whether the client needs to update the cache.
      * @param serverCache the admin local cache
-     * @param clientMd5 the client md5 value
+     * @param clientHashValue the client hashValue
      * @param clientModifyTime the client last modify time
      * @return true: the client needs to be updated, false: not need.
      */
-    private boolean checkCacheDelayAndUpdate(final ConfigDataCache serverCache, final String clientMd5, final long clientModifyTime) {
+    private boolean checkCacheDelayAndUpdate(final ConfigDataCache serverCache, final int clientHashValue, final long clientModifyTime) {
         // is the same, doesn't need to be updated
-        if (StringUtils.equals(clientMd5, serverCache.getMd5())) {
+        if (clientHashValue != serverCache.getHashValue()) {
```

Comment:
?

Reply:
However, in terms of performance  
sha1 < md5 < hash ; 
length:
sha1> md5 > hash;
so why use sha1 replace md5?

Reply:
sorry,